test/modules/generic-mqtt-tester/src/message_initiator.rs

```diff
@@ -59,11 +51,13 @@ impl MessageInitiator {
         let mut publish_handle = self.publish_handle.clone();
         loop {
             info!("publishing message {} to upstream broker", seq_num);
+            let dummy_payload = "a".repeat(self.settings.message_size_in_bytes() as usize);
```

Comment:
if message_size_in_bytes is constant for the duration of a test, do you want to move it out of the loop?